ACE/NEWS

```diff
@@ -1,6 +1,8 @@
 USER VISIBLE CHANGES BETWEEN ACE-6.5.8 and ACE-6.5.9
 ====================================================
 
+. _WIN32_WINNT updated to 0x0600 from 0x0501.
```

Comment:
We should add some more information about the implications for the user for this

Reply:
I reconsidered this one...
Since we are changing this in config-win32-common, this seems like a good time to either:
- not change _WIN32_WINNT at all, this should be up to the user
- not reduce _WIN32_WINNT if the user has already set it to a larger value
- possibly emit a warning if _WIN32_WINNT is too low
In any case, it seems like this is a good time to make ACE play better with other libs/configurations